From: Wei Yongjun <yongjun_wei@trendmicro.com.cn>
Date: Tue, 4 Dec 2012 00:15:42 -0500
Patch-mainline: v3.9-rc1
Subject: Drivers: hv: remove unused variable in vmbus_recvpacket_raw()
Git-commit: f994a154c5318d132200862686fde13dca0979b1

The variable userlen is initialized but never used
otherwise, so remove the unused variable.

Signed-off-by: Wei Yongjun <yongjun_wei@trendmicro.com.cn>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Acked-by: <ohering@suse.de>
---
 drivers/hv/channel.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/hv/channel.c b/drivers/hv/channel.c
index 064257e..0b122f8 100644
--- a/drivers/hv/channel.c
+++ b/drivers/hv/channel.c
@@ -780,7 +780,6 @@ int vmbus_recvpacket_raw(struct vmbus_channel *channel, void *buffer,
 {
 	struct vmpacket_descriptor desc;
 	u32 packetlen;
-	u32 userlen;
 	int ret;
 	bool signal = false;
 
@@ -795,7 +794,6 @@ int vmbus_recvpacket_raw(struct vmbus_channel *channel, void *buffer,
 
 
 	packetlen = desc.len8 << 3;
-	userlen = packetlen - (desc.offset8 << 3);
 
 	*buffer_actual_len = packetlen;
 
